src/components/Results/Result/Result.js

```diff
@@ -2,34 +2,35 @@ import React from 'react';
 
 import './Result.css';
 
-const cardStyle = {
-    width: '10rem',
-    maxHeight: '18rem'
-    
-};
 
-const imageStyle = {
-    minWidth: '6em',
-    maxWidth: '10em',
-    maxHeight: '12em',
-    height: 'auto',
-    vertialAlign: 'center'
-};
+export default function Result(props) {
 
-const bodyStyle = {
-    padding: '0.25rem'
-};
+    const cardStyle = {
+        width: '10rem',
+        maxHeight: '18rem'
+        
+    };
+    
+    const imageStyle = {
+        minWidth: '6em',
+        maxWidth: '10em',
+        maxHeight: '12em',
+        height: 'auto',
+        vertialAlign: 'center'
+    };
+    
+    const bodyStyle = {
+        padding: '0.25rem'
+    };
```

Comment:
We can move the JSX style variables into the associated CSS files, but let's do that in a follow-up PR after merging this to avoid conflicts

Reply:
Sounds good!